srcpkgs/deheader/template

```diff
@@ -1,18 +1,17 @@
 # Template file for 'deheader'
 pkgname=deheader
-version=1.6
-revision=4
-create_wrksrc=yes
+version=1.7
+revision=1
 hostmakedepends="tar"
-depends="python"
+depends="python3"
 short_desc="C and C++ header analyzer"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="BSD-2-Clause"
 homepage="http://www.catb.org/~esr/deheader/"
 distfiles="http://www.catb.org/~esr/deheader/${pkgname}-${version}.tar.gz"
-checksum=3b99665c4f0dfda31d200bf2528540d6898cb846499ee91effa2e8f72aff3a60
+checksum=6856e4fa3efa664a0444b81c2e1f0209103be3b058455625c79abe65cf8db70d
 skip_extraction="${pkgname}-${version}.tar.gz"
```

Comment:
It looks like this, the entire `do_extract` function below the `hostmakedepends="tar"` are now unnecessary. Please confirm this to be true and, if so, remove them.

Reply:
Seems, correct. ls -l doesn't show man page is linked, and man deheader works fine. So fix incoming.